ftw/file/testing.py

```diff
@@ -31,10 +31,16 @@ def setUpZope(self, app, configurationContext):
             '</configure>',
             context=configurationContext)
 
+        import plone.app.imaging
+        xmlconfig.file('configure.zcml', plone.app.imaging,
```

Comment:
`plone.app.imaging` has a `z3c.autoinclude` entry point, it *should*  already be loaded.
Is there a reason for this or can we remove it?